fix #1541: Improve deprecation messages (#1548)
vuejs/vue-test-utils: docs/api/wrapper/isVueInstance.md

## isVueInstance

::: warning Deprecation warning
`isVueInstance` is deprecated and will be removed in future releases.

> Most of the times, tests relying on `isVueInstance` provide little to no-value so you could simply remove them.

Reviewer: Technically those are only valid in shallowMount environments, to assert dynamic components via `<component :is=""`, nothing much else.... I had to use it probably once or twice.

The proposed solution does the job :)

> Most of the times, tests relying on `isVueInstance` provide little to no-value so you could simply remove them.

Reviewer 2: ```suggestion
Tests relying on the `isVueInstance` assertion provide little to no value. We suggest replacing them with better assertions.
```

Active voice

Reviewer 2: Even if you don't take the suggestion, "Most of the times" should be "Most of the time"

> If you want to keep these tests, a valid replacement is to assert that `wrapper.find(...).vm` exists.

Reviewer 2: ```suggestion
To keep these tests, a valid replacement for `isVueInstance()` is a truthy assertion of `wrapper.find(...).vm` 
```
